plugins/woocommerce/tests/e2e-pw/tests/shopper/cart-block-calculate-shipping.spec.js

```diff
@@ -137,12 +137,21 @@ test.describe( 'Cart Block Calculate Shipping', () => {
 		await addAProductToCart( page, product1Id );
 		await page.goto( cartBlockPage.slug );
 
+		await page
+			.locator( 'button', {
+				hasText: 'Enter the address to check delivery options',
+			} )
+			.click();
+
 		// Set shipping country to Netherlands
-		await page.getByLabel( 'Add an address for shipping' ).click();
 		await page.getByRole( 'combobox' ).first().fill( 'Netherlands' );
 		await page.getByLabel( 'Postal code' ).fill( '1011AA' );
 		await page.getByLabel( 'City' ).fill( 'Amsterdam' );
-		await page.getByRole( 'button', { name: 'Update' } ).click();
+		await page
+			.locator(
+				'.wc-block-components-shipping-calculator-address__button'
+			)
+			.click();
```

Comment:
See my previous comment regarding using `page.getByRole()` instead of `page.locator()`.

Reply:
@nielslange I initially tried using the `page.getByRole()`. However, it won't work even if the button is visible on the page. I'd really appreciate it if you could guide me what's the best way to use the `page.getByRole()` for the below button:
```html
<button type="submit" class="wc-block-components-button wp-element-button wc-block-components-shipping-calculator-address__button outlined">
  <span class="wc-block-components-button__text">Check delivery options</span>
</button>
```